### ISSUE TITLE
Using hydrate.renderToString produces "invalid" HTML with <track>

### ISSUE THREAD

**Issue by JaapBarnhoorn** [ionic-team/stencil#2994]
**Stencil version:**

```
@stencil/core@2.6.0 
```

**I'm submitting a:**

[x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://stencil-worldwide.herokuapp.com/ or https://forum.ionicframework.com/

**Current behavior:**
The resulting html of using `hydrate.renderToString()` is no valid html.

When providing it with a `<track>` element, it will close the tag and produces `<track></track>`
This generates errors in our WCAG report.

**Expected behavior:**
It should just leave `<track>` how it is:
https://www.w3.org/TR/html52/semantics-embedded-content.html#the-track-element

**Steps to reproduce:**

If you use below html as a source and hydrate it with renderToString() it will produce the closing `</track>` tag.
`<source>` seems to work fine

**Related code:**

```html
<video>
	<source src="x" type="video/mp4">
	<track
		kind="subtitles"
		src="x"
	>
</video>
```

**Other information:**

**Comment by splitinfinities** [comment]
Hey there, thank you for the patience getting back to you. The new team is getting started and we're working through the backlog now.

Thanks for adding this. We don't have a timeframe, but we have identified the issue.

**Comment by rwaskiewicz** [comment]
👋 

Thanks for the issue report @JaapBarnhoorn! I've labeled this to be properly ingested by our internal issue tracker for the team to take a closer look!

**Comment by JaapBarnhoorn** [comment]
Thank you @rwaskiewicz.

**Comment by rwaskiewicz** [comment]
The fix for this PR was included in today's [v4.18.0 release](https://github.com/ionic-team/stencil/releases/tag/v4.18.0)
